package.json

```diff
@@ -38,7 +38,7 @@
     "extend": "^3.0.2",
     "gcp-metadata": "^2.0.0",
     "parse-duration": "^0.1.1",
-    "pprof": "^1.0.0",
+    "pprof": "1.1.0",
```

Comment:
@ofrobots for Ali's opinion.

I think even with pinning it's common to still allow revision version changes so that security patches can still be picked up.